contracts/token/ERC1155/ERC1155.sol

```diff
@@ -165,25 +218,9 @@ contract ERC1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
         bytes memory data
     ) internal virtual {
```

Comment:
As in ERC20, the transfer/burn/mint functions that call into `_update` shouldn't be virtual.